src/many-abci/src/main.rs

```diff
@@ -73,6 +75,11 @@ struct Opts {
     /// is given.
     #[clap(long, short)]
     migrations_config: Option<PathBuf>,
+
+    /// Database path to the cache. If unspecified, the server will not
+    /// verify transactions for duplicate requests.
+    #[clap(long)]
+    cache_db: Option<PathBuf>,
```

Comment:
This should be mandatory in `many-abci`. Not using a cache will result in potential duplicate transactions. It is dangerous to make it optional.

Reply:
Done.